README.md

```diff
@@ -1,25 +1,26 @@
 <a href="https://opensource.newrelic.com/oss-category/#new-relic-experimental"><picture><source media="(prefers-color-scheme: dark)" srcset="https://github.com/newrelic/opensource-website/raw/main/src/images/categories/dark/Experimental.png"><source media="(prefers-color-scheme: light)" srcset="https://github.com/newrelic/opensource-website/raw/main/src/images/categories/Experimental.png"><img alt="New Relic Open Source experimental project banner." src="https://github.com/newrelic/opensource-website/raw/main/src/images/categories/Experimental.png"></picture></a>
 
-# Go Easy Instrumentation [![codecov](https://codecov.io/gh/newrelic/go-easy-instrumentation/graph/badge.svg?token=0qFy6WGpL8)](https://codecov.io/gh/newrelic/go-easy-instrumentation)
-Go is a compiled language with an opaque runtime, making it unable to support automatic instrumentation like other languages. For this reason, the New Relic Go agent is designed as an SDK. Since the Go agent is an SDK, it requires more manual work to set up than agents for languages that support automatic instrumentation. 
+# Go instrumentation tool
```

Comment:
```suggestion
# Go Easy Instrumentation [![codecov](https://codecov.io/gh/newrelic/go-easy-instrumentation/graph/badge.svg?token=0qFy6WGpL8)](https://codecov.io/gh/newrelic/go-easy-instrumentation)
```

We want `Go Easy Instrumentation` to be the name of the product for now, and I would like the `codecov` badge displayed to show that this is a dependable product.

Reply:
@iamemilio on the main tech docs site, I just pushed up the label "Go easy instrumentation tool" with lower case to follow our style guide. Can we do that here, too?

I'm not sure how to set the code coverage to "dependable." It looks like that would require a custom badge. Can you figure that out?

Reply:
Hi, yes. You can just restore the badge as it was: `[![codecov](https://codecov.io/gh/newrelic/go-easy-instrumentation/graph/badge.svg?token=0qFy6WGpL8)](https://codecov.io/gh/newrelic/go-easy-instrumentation)`

Having coverage above 80% is considered strong coverage.